qdev_wrappers/tests/dataset/test_doNd.py

```diff
@@ -165,8 +164,8 @@ def test_do1d_output_data(_param, _param_set):
     data = load_by_id(exp[0])
 
     assert data.parameters == f'{_param_set.name},{_param.name}'
-    assert data.get_values(_param.name) == [[1]] * 5
-    assert data.get_values(_param_set.name) == [[0], [0.25], [0.5], [0.75], [1]]
+    assert np.allclose(data.get_parameter_data(_param.name)[_param.name][_param.name], np.array([1]*5))
```

Comment:
Slightly cleaner way to write `np.array([1]*5))` with numpy would be `np.ones(5)`

Reply:
Good point, thanks. I will make the change accordingly and push it in a moment.